Jenkinsfile

```
@@ -1140,11 +1140,12 @@ pipeline {
                                   rm -rf "Functional/"
                                   mkdir "Functional/"
                                   # compress those potentially huge DAOS logs
-                                  if daos_logs=$(ls install/lib/daos/TESTING/ftest/avocado/job-results/*/daos_logs/*); then
+                                  if daos_logs=$(find install/lib/daos/TESTING/ftest/avocado/job-results/*/daos_logs/* -maxdepth 0 -type f -size 2M); then
                                       lbzip2 $daos_logs
                                   fi
```

You could simplify this a bit but more importantly you want **+** before the `2M`:
```suggestion
                                  find install/lib/daos/TESTING/ftest/avocado/job-results/*/daos_logs/* -maxdepth 0 -type f -size +2M -print0 | 
                                      xargs -0r lbzip2
```

---

Ah. -size 2M catches files that are larger than 1M, +1M is probably the same but clearer as you say.

---

I believe `-size 2M` catches files that are exactly 2M and no others.  None bigger or smaller.

---

I think it was probably doing it in buckets, so catching files between 2M and 3M in size..